### PR TITLE
rsx: Add some sensible fallbacks

### PR DESCRIPTION
**Pull request by kd-11**
Fixes https://github.com/RPCS3/rpcs3/issues/11218
Fixes https://github.com/RPCS3/rpcs3/issues/11216

### CODE REVIEW

File: rpcs3/Emu/RSX/GL/GLPresent.cpp [hunk 1]
```diff
@@ -141,6 +141,11 @@ void GLGSRender::flip(const rsx::display_flip_info_t& info)
 		buffer_width = std::min(buffer_width, avconfig.resolution_x);
 		buffer_height = std::min(buffer_height, video_frame_height);
 	}
+	else if (!buffer_width)
```
Comment (elad335) [inline rpcs3/Emu/RSX/GL/GLPresent.cpp:144]: std::min is used above, if it can be 0 it will also yield zero.

File: rpcs3/Emu/RSX/rsx_utils.cpp [hunk 2]
```diff
@@ -135,6 +135,20 @@ namespace rsx
 		return result;
 	}
 
+	avconf::avconf()
```
Comment (elad335) [inline rpcs3/Emu/RSX/rsx_utils.cpp:138]: noexcept please.

File: rpcs3/Emu/RSX/GL/GLPresent.cpp (hunk 1 above)
Comment (kd-11) [reply rpcs3/Emu/RSX/GL/GLPresent.cpp:144]: It cannot be zero.

Comment (elad335) [reply rpcs3/Emu/RSX/GL/GLPresent.cpp:144]: cellVideoOutConfigure is not calling sys_rsx_context_attribute, display_buffers data may not be initailized even if OutConfigure was called,

Comment (elad335) [reply rpcs3/Emu/RSX/GL/GLPresent.cpp:144]: i.e. `buffer_height = std::min(buffer_height, video_frame_height)` will result in 0 if it sys_rsx syscalll was not called.

**Pull request merged**